src/debug/debug_impl.ts

```diff
@@ -181,20 +181,33 @@ export class DartDebugSession extends DebugSession {
 		if (!breakpoints)
 			breakpoints = [];
 
-		let sourcePath = source.path;
-		let uri = this.packageMap.convertFileToPackageUri(sourcePath);
-		if (!uri)
-			uri = fileToUri(sourcePath);
-
-		this.threadManager.setBreakpoints(uri, breakpoints).then((result: boolean[]) => {
-			let bpResponse = [];
-			for (let verified of result) {
-				bpResponse.push({ verified: verified });
-			}
+		let uris = this.getBreakpointUris(source.path);
 
-			response.body = { breakpoints: bpResponse };
-			this.sendResponse(response);
-		}).catch((error) => this.errorResponse(response, `${error}`));
+		uris.forEach(uri => {
```

Comment:
You may want to add some quick docs here to say why we're doing this (to support both file: and package: uris at runtime, for the same file).